ui/ts/pages/nodes.ts

```diff
@@ -322,6 +330,7 @@ module AdminViews {
                 .nonNegativeRate()
                 .title("DDL Statements")
             ).format(d3.format(".1")).title("Schema Changes")
+              .tooltip("The number of DDL statements across all nodes, averaged over a 10 second period.")
```

Comment:
What happens to these tooltips when the time interval is changed? Seems we should remove the "over a 10 second period" suffix and add it dynamically based on the current time interval.